.openshiftio/application.yaml

```diff
@@ -32,30 +40,24 @@ parameters:
   from: '[a-zA-Z0-9]{40}'
   generate: expression
 objects:
-- apiVersion: v1
+- apiVersion: image.openshift.io/v1
```

Comment:
Why do this if it's not done for other resources?

Reply:
I have made it v1